### PR TITLE
Fixed naming and small syntax issues for new services

### PR DESCRIPTION
**Pull request by KiLanger**

### CODE REVIEW

**Review by delvh — APPROVED** [review]

File: docs/docs/dev-manuals/services/payment.mdx [hunk 1]
```diff
@@ -8,7 +8,7 @@ It _is not a payment provider_ but simulates communicating with one via the `Sim
 
 ## Domain Model
 
-CustomImage path="/renderedDiagrams/inventoryDomainModel" width="1542.5" height="663.25" />
+<CustomImage path="/renderedDiagrams/paymentDomainModel" width="1542.5" height="663.25" />
```
Comment (nk-coding) [inline docs/docs/dev-manuals/services/payment.mdx]: are you sure these dimensions are correct?

Comment (nk-coding) [reply docs/docs/dev-manuals/services/payment.mdx]: even for inventory they do not seem to be 100% correct, maybe you can also fix this with this PR

Comment (KiLanger) [reply docs/docs/dev-manuals/services/payment.mdx]: I copied them from somewhere else. It's kind of hard to test without being able to compile the docs :/

Comment (nk-coding) [reply docs/docs/dev-manuals/services/payment.mdx]: you don't need to compile the docs
you just need to render the diagrams (either using the cli command or using the hylimo web UI) and then look at the svg to copy the width and height

**Review by nk-coding — APPROVED** [review]

**Pull request merged**